models/qualtrics__response_breakdown.sql

```diff
@@ -1,5 +1,5 @@
 with response as (
-    -- will probably move the logic from this int model to the end model directly 
+    -- will probably move the logic from this int model to the end model directly  
```

Comment:
I think it is fine to keep as intermediate model

Reply:
cool yah if int models are ephmeral this doesn't matter too much